src/Config.php

```diff
@@ -1101,133 +1038,35 @@ public static function showSystemInfoTable($params = [])
             echo "</td></tr>";
         }
 
-        echo "<tr class='tab_bg_2'><th>" . __('To copy/paste in your support request') . "</th></tr>\n";
-
         echo "</table>";
+
+        TemplateRenderer::getInstance()->display('pages/setup/general/systeminfo_table.html.twig', [
+            'ver' => $ver,
+            'language' => $oldlang,
+            '_server' => $_SERVER,
+            'db_info' => $DB->getInfo(),
+            'core_requirements' => $requirements,
+            'system_info_objs' => $system_info_objs,
+            'locales_overrides' => $files
```

Comment:
Lines 1032 to 1038 should probably be removed, as displayed information are also present in Twig template.